components/symbl_ai/actions/get-conversations/get-conversations.mjs

```diff
@@ -0,0 +1,81 @@
+import symblAIApp from "../../symbl_ai.app.mjs";
+
+export default {
+  key: "symbl_ai-get-conversations",
+  name: "Get Conversations",
+  description: "Get a list of all conversations. See the doc [here](https://docs.symbl.ai/docs/conversation-api/all-conversations/)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    symblAIApp,
+    limit: {
+      type: "integer",
+      label: "Limit",
+      description: "Specifies the limit number of conversations to be returned. Default value is `20`.",
+      optional: true,
+      min: 0,
+      max: 65536,
+    },
+    offset: {
+      type: "integer",
+      label: "Offset",
+      description: "Specifies the number of items to skip before applying `limit`",
+      optional: true,
+    },
+    order: {
+      type: "string",
+      label: "Order",
+      description: "Specifies the order in which the results should be sorted based on the start time.",
+      options: [
+        "asc",
+        "desc",
+      ],
+      optional: true,
+    },
+    startTime: {
+      type: "string",
+      label: "Start Time",
+      description: "Specifies the start of the date and time range for the results to be returned.",
```

Comment:
I think it would be better to inform users the acceptable value type

```suggestion
      description: "Specifies the start of the date and time range for the results to be returned. Values accepted are [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) formatted stringsValues accepted are [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) formatted strings",
```

Reply:
Addressed in 4354697b.